components/centraldashboard/public/components/dashboard-view.js

```diff
@@ -107,6 +107,11 @@ export class DashboardView extends PolymerElement {
                         desc: 'Katib',
                         link: `/katib/`,
                     },
+                    {
+                        text: 'View Metadata Artifacts',
+                        desc: 'Artifact Store',
+                        link: `/metadata/`,
+                    },
```

Comment:
What's the point of having the same link twice? Maybe a more helpful link would be a docs page for artifacts in getting started?